app/views/collections/_collection_blurb.html.erb

```diff
@@ -2,7 +2,7 @@
 <li class="<% if collection.user_is_owner?(current_user) %>own <% end %>collection picture blurb group" role="article">
   <% logged_in = (logged_in_as_admin? || logged_in?) ? "logged-in" : "logged-out" %>
   <% # remember to update collection_sweeper if you change this key %>
-  <% cache("collection-blurb-#{logged_in}-#{collection.id}-v3", skip_digest: true) do %>
+  <% cache("collection-blurb-#{logged_in}-#{collection.id}-v3", expires_in: ArchiveConfig.MINUTES_UNTIL_COLLECTION_BLURBS_EXPIRE.minutes, skip_digest: true) do %>
```

Comment:
Since has `skip_digest: true`, I think we'll want to update the version of this key too; otherwise, the whole cache might (?) need to be cleared for changes to show up consistently